test/server/config_validation/BUILD

```diff
@@ -1,6 +1,17 @@
 licenses(["notice"])  # Apache 2
 
-load("//bazel:envoy_build_system.bzl", "envoy_cc_test", "envoy_package")
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_fuzz_test",
+    "envoy_cc_test",
+    "envoy_cc_test_library",
+    "envoy_package",
+    "envoy_select_hot_restart",
```

Comment:
A number of these are unnecessary; you only added a `envoy_cc_fuzz_test` target, so this should be the only new one needed.